lms/djangoapps/email_marketing/migrations/0006_auto_20170711_0615.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('email_marketing', '0005_emailmarketingconfiguration_user_registration_cookie_timeout_delay'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='emailmarketingconfiguration',
+            name='user_registration_cookie_timeout_delay',
+            field=models.FloatField(default=3.0, help_text='The number of seconds to delay/timeout wait to get cookie values from sailthru.'),
```

Comment:
Yes this is for new entries. I have already changed timeout value to 3 in prod via admin.